modules/admin_manual/pages/configuration/user/oidc/kopano-setup.adoc

```diff
@@ -0,0 +1,105 @@
+= Example Setup Using Kopano Konnect
+:toc: right
+:toclevels: 2
+:openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
+:konnect: https://github.com/Kopano-dev/konnect
+:konnect-docs: https://github.com/Kopano-dev/konnect#running-konnect
+:konnect-webserver: https://documentation.kopano.io/kopanocore_administrator_manual/configure_kc_components.html#configure-a-webserver-for-konnect
+
+== Introduction
+
+Kopano Konnect is an OpenID provider (IdP) that directly integrates a Web login and consent form. It brings support for both OpenID Connect (OIDC) and Open Authentication (OAuth 2.0). In addition to the easier integration with third-party applications, Kopano Konnect will also provide the authentication part for the Kopano RestAPI and clients consuming it.
```

Comment:
Please make sure to use 'OpenID Connect' and not 'OpenID'
These are two totally different things

Reply:
Thanks, will do asap

Reply:
fixed